packages/editor/src/store/effects/utils.js

```diff
@@ -13,21 +13,5 @@ import { select, subscribe } from '@wordpress/data';
  * @return {Promise} Selector result.
  */
 export function resolveSelector( namespace, selectorName, ...args ) {
-	return new Promise( ( resolve ) => {
-		const hasFinished = () => select( 'core/data' ).hasFinishedResolution( namespace, selectorName, args );
-		const getResult = () => select( namespace )[ selectorName ].apply( null, args );
```

Comment:
I think we should just replace the usage directly in this case, and in the widgets package too.